contracts/migrations/2_deploy_v0.js

```diff
@@ -24,8 +35,10 @@ async function deploy(options, accounts) {
     { from: accounts[0] }
   );
 
+  let noLossDaoInstance = await NoLossDao_v0.at(noLossDao.address);
+
   // TODO: fix the interval here!
-  await noLossDao.initialize(poolDeposits.address, '1800', {
+  await noLossDaoInstance.initialize(poolDeposits.address, '1800', {
```

Comment:
The risk of someone initializing this before us is neglegable, and if it happens then we can just redeploy. No need to mess around with create 2 or whatever other strategies to make deployment atomic!

Reply:
Okay this makes sense!